packages/core/src/linker/ng_module_factory.ts

```diff
@@ -7,7 +7,7 @@
  */
 
 import {Injector} from '../di/injector';
-import {EnvironmentInjector} from '../di/r3_injector';
+import {EnvironmentInjector, R3Injector} from '../di/r3_injector';
```

Comment:
Q: Isn't this a leftover ? - I do not see where `R3Injector` is being used in this file 

```suggestion
import {EnvironmentInjector} from '../di/r3_injector';
```